os/drivers/wireless/bluetooth/bt_null.c

```diff
@@ -0,0 +1,310 @@
+/****************************************************************************
+ * drivers/wireless/bluetooth/bt_null.c
+ * UART based Bluetooth driver
```

Comment:
samsung license is required. and if source is not come from nuttx. I think you don't need to write nuttx license